cluster/src/main/java/org/apache/iotdb/cluster/query/ClusterPlanExecutor.java

```diff
@@ -307,7 +307,10 @@ protected int getNodesNumInGivenLevel(PartialPath path, int level) throws Metada
     // if path doesn't end with **.
     // e.g. we have SG root.sg.a and root.sg.b, the query path is root.sg, we should return the map
     // with key root.sg.a and root.sg.b instead of an empty one.
-    PartialPath wildcardPath = path.concatNode(IoTDBConstant.MULTI_LEVEL_PATH_WILDCARD);
+    PartialPath wildcardPath = path;
+    if (!wildcardPath.getMeasurement().equals(IoTDBConstant.MULTI_LEVEL_PATH_WILDCARD)) {
+      wildcardPath = wildcardPath.concatNode(IoTDBConstant.MULTI_LEVEL_PATH_WILDCARD);
+    }
```

Comment:
Thanks for your job, however, I have one issue, It's a bug or not？if so, why not add one UT to cover it?

Reply:
`IoTDBMetadataFetchIT.showCountTimeSeries` can test this in the cluster test. I have just found that the ClusterPlanExecutor has its own UT file. And I will try to add UTs in following PRs.